pages/[network]/account/payments.tsx

```diff
@@ -66,7 +66,7 @@ export default function Payments() {
   const {getPayments} = useApi()
   const {wallet} = useAuthentication()
   const [payments, setPayments] = useState<Payment[]>([]);
-  const [hasMore, setHasMore] = useState(false);
+  const [hasMore] = useState(false);
```

Comment:
do we ever change "hasMore"? If not -- is it really needed?

Reply:
we still don't use hasMore, but we will use it to paginate the payments, I removed the 'set' temporarily to correct the tsconfig warnings.